actions/post_actions.jsx

```diff
@@ -185,6 +185,7 @@ export function pinPost(postId) {
         if (rhsState === RHSStates.PIN) {
             addPostToSearchResults(postId, state, dispatch);
         }
+        return {data: true};
```

Comment:
are these changes related to apps?

Reply:
In order to be recognized in Typescript as Action func, they cannot return void, they must return either {data: any} or {error: any}.